packages/flutter/lib/src/widgets/platform_view.dart

```diff
@@ -199,6 +200,8 @@ class AndroidView extends StatefulWidget {
 ///
 /// Construction of UIViews is done asynchronously, before the UIView is ready this widget paints
 /// nothing while maintaining the same layout constraints.
+/// 
+/// If a conic path clipping is applied to a UIKitView, a quad path is used to approximate the clip due to limitation of Quartz.
```

Comment:
nit: we try to keep doc comment lines to be ~80 characters long. This one locks longer.

Reply:
It's definitely longer :), I will edit before merging.